docs/Getting_Started.md

```diff
@@ -0,0 +1,437 @@
+
+And here's what the finished app will look like:
+
+![Demo of example app](getting-started-demo.gif)
+
+To make the user flow a bit more explicit:
+1. User 1 initiates a new game and picks their object using the app’s `/challenge` slash command
+2. A message is sent to channel with a button inviting others to accept the challenge
+3. User 2 presses the **Accept** button
+4. User 2 is sent an ephemeral message where they select their object of choice
+5. The result of the game is posted back into the original channel for all to see
+
+## Creating an app
+
+The first thing we’ll need to do is create an app. Navigate to [the developer dashboard](https://discord.com/developers/applications), then click **New Application** in the upper right corner.
+
+![App creation modal](app-create-modal.png)
+
+Enter a name for your app, then click **Create**.
+
+Once you create an app, you'll land on the **General Overview** page of the app's settings. Here you can view and configure basic information about the app, like its description and icon. You’ll also see an **Application ID** and **Interactions Endpoint URL**, which we’ll use a bit later in the guide.
+
+### Configuring a bot
+
+Next we'll add a bot user to your app, which allows it to appear in Discord similar to other members. On the left hand sidebar click **Bot**, then the **Add Bot** button.
+
+Once you create a bot, you’ll have an option to update its icon and username. Under that, there’s a **Token** section with a **Reset Token** button.
+
+![Bot tab in app settings](app-add-bot.png)
+
+Bot tokens are used to authorize API requests and carry all of your bot user’s permissions, making them *highly sensitive*. Make sure to *never* share your token or check it into any kind of version control.
+
+Go ahead and click **Reset Token**, and store the token somewhere safe (like in a password manager).
+
+> warn
+> You won’t be able to view your token again unless you regenerate it, so make sure to keep it somewhere safe. 
+
+### Adding permissions
+
+Apps need approval to perform actions inside of Discord (like installing a slash command or adding emojis). So before installing your app, let's add some permissions. Click on **OAuth2** in the left sidebar, then **URL generator**.
+
+> info
+> The URL generator helps create an installation link with the permissions your app needs to function. You can use the link to install the app onto your server, or share it with others so they can install it on their own.
+
+For now, we’ll just add a couple of permissions:
+- `applications.commands` lets your app create commands in guilds its installed
+- `bot` is to enable your bot user. After you click bot, you can also add individual permissions. For now, just check **Send Messages**.
+
+![URL generator screenshot](url-generator.png)
+
+Once you add permissions, you should see a URL that you can copy to install your app.
```

Comment:
"Permissions" is being used for two meanings here, scopes _and_ role permissions, which makes it a bit confusing for newer devs